### ISSUE TITLE
Anyone succeed to infer with Inception_v3 or ShuffleNet_v2 ?

### ISSUE THREAD

**Issue by matfeb** [alibaba/MNN#597]
Hello there,

MobileNet_V2 and SqueezeNet_v1.1 infer well on Android but I can't infer with other models so far.

I tried with ONNX -> MNN conversion, TFLite --> MNN and even PyTorch --> ONNX --> MNN.

Is there any parameters to add for specific models ? 
For example, I know that Inception needs a 299*299 image but which synset_words, which mConfig values, which NCHW values ?

Some models does not even load with SG_FAULT.

I'm starting to think that MNN on Android is only working with MobileNet and SqueezeNet, anyway there is not enough documentation to load another model. It's just a kind of "Convert the model and load it" without further instructions.

Thanks for your advice.

**Comment by jxt1234** [comment]
You can use the inception model from https://www.yuque.com/mnn/en/demo_zoo.

**Comment by jxt1234** [comment]
Caffe and tensorflow model (pb) is supported better.
If trained from tensorflow, don't turned to tflite, just use pb.

**Comment by matfeb** [comment]
Thanks for your reply

> You can use the inception model from https://www.yuque.com/mnn/en/demo_zoo.

I've always taken models from your page or tflite models from tensorflow.
Unfortunately it does not seem to work. 
There is no execution problem but it does not work as mobileNet or squeezeNet.
I don't know how to explain. Inference result is not accurate and very slow.

> Caffe and tensorflow model (pb) is supported better.
> If trained from tensorflow, don't turned to tflite, just use pb.

Good to know, I have to find tensorflow pretrained models now
Thanks.

**Comment by jxt1234** [comment]
I use inception_v3's tflite model and it's ok . You can update the code to solve tflite convert bug

**Comment by matfeb** [comment]
Hey,

I tried again. I've converted tflite model to mnn, there is a softmax conversion. If I try with the tf model there is a reshape.

Anyway I have the same log : "VideoActivity: session result too big (1001), model incorrect ?" on both models.

I've used : 

- label.txt from the model

- 299*299 image resize.

- dimensions[0] = 1 not NCHW

- there is no config.mean nor config.normal variable

**Comment by jxt1234** [comment]
It seems like the error of imageprocess's config. 
config.mean = new float[]{103.94f, 116.78f, 123.68f};
config.normal = new float[]{0.017f, 0.017f, 0.017f};

You can refer to the code of demo/exec/pictureRecognition.cpp
I use the pictureRecognition demo and Inception_v3 works ok
